Crashlytics/Crashlytics/Operations/Symbolication/FIRCLSThreadArrayOperation.m

```diff
@@ -14,13 +14,13 @@
 
 #import "FIRCLSThreadArrayOperation.h"
 
-#import "FIRCLSStackFrame.h"
+#import "FIRStackFrame_Private.h"
```

Comment:
Double check all the imports of the private header. Some of them are not needed or import the public header when the code does not access any of the private members.